package.json

```diff
@@ -33,6 +33,7 @@
     "dependencies": {
         "jquery": "2.2.1",
         "angular2": "2.0.0-beta.12",
+        "ng2-translate": "^1.11.0",
```

Comment:
Please do not use '^' for production dependencies. It include also minor updates which could be very risky and lead maybe to pull breaking changes in without knowing.